src/main/anovos/data_analyzer/quality_checker.py

```diff
@@ -51,12 +51,16 @@ def duplicate_detection(spark, idf, list_of_cols='all', drop_cols=[], treatment=
     odf_tmp = idf.drop_duplicates(subset=list_of_cols)
     odf = odf_tmp if treatment else idf
 
-    odf_print = spark.createDataFrame([["rows_count", idf.count()], ["unique_rows_count", odf_tmp.count()]],
-                                      schema=['metric', 'value'])
-
+    odf_print = spark.createDataFrame([["rows_count", float(idf.count())], \
+					["unique_rows_count", float(odf_tmp.count())], \
+					["duplicate_rows", float(idf.count() - odf_tmp.count())], \
+					["pct_duplicate_rows", round((idf.count() - odf_tmp.count())/idf.count(), 4)]], \
+					schema=['metric', 'value'])
     if print_impact:
         print("No. of Rows: " + str(idf.count()))
         print("No. of UNIQUE Rows: " + str(odf_tmp.count()))
+        print("No. of Duplicate Rows: " + str(odf_tmp.count() - idf.count()))
+        print("Percentage of Duplicate Rows: " + str((odf_tmp.count() - idf.count())/idf.count()))
```

Comment:
- Round off to be consistent with output dataframe. 
- Docstring to be updated wrt metric dataframe
- duplicate_pct instead of pct_duplicate_rows? similar to mode_rows & mode_pct?

Reply:
Basic report generation needs to be updated too.